src/aks-preview/HISTORY.rst

```diff
@@ -9,6 +9,13 @@ If there is no rush to release a new version, please just add a description of t
 
 To release a new version, please select a new version number (usually plus 1 to last patch version, X.Y.Z -> Major.Minor.Patch, more details in `\doc <https://semver.org/>`_), and then add a new section named as the new version number in this file, the content should include the new modifications and everything from the *Pending* section. Finally, update the `VERSION` variable in `setup.py` with this new version number.
 
+1.0.0b6
```

Comment:
please keep `Pending` at top and move `* Vendor new SDK and bump API version to 2023-11-02-preview.` to 1.0.0b6